src/vogleditor/vogleditor.cpp

```diff
@@ -471,7 +471,7 @@ VoglEditor::Prompt_Result VoglEditor::prompt_generate_trace()
 #endif
 
             QString cmdLine = m_pLaunchTracerDialog->get_command_line();
-            QProcessEnvironment env = m_pLaunchTracerDialog->get_process_environment();
+            QProcessEnvironment env = m_pLaunchTracerDialog->get_process_environment (sizeof (void *) < 8);
```

Comment:
Absolutely.  The problem is, leaving this as-is was breaking the tracing in my case.